README.md

```diff
@@ -3,587 +3,21 @@
-      "allocatedReactivations": {
-        "currentPeriodReactivation": 2
-      }
-    }, {
-      "companyName": "Company Name ltd",
-      "utr": "1123456789",
-      "consenting": true,
-      "qicElection": true,
-      "netTaxInterestExpense": 0,
-      "netTaxInterestIncome": 50,
-      "taxEBITDA": 5,
-      "allocatedReactivations": {
-        "currentPeriodReactivation": 2
-      }
-    }],
-    "angie": 1.11,
-    "returnContainsEstimates": false,
-    "groupSubjectToInterestRestrictions": false,
-    "groupSubjectToInterestReactivation": true,
-    "totalRestrictions": 0,
-    "groupLevelAmount": {
-      "interestReactivationCap": 300,
-      "interestAllowanceBroughtForward": 3,
-      "interestAllowanceForPeriod": 4,
-      "interestCapacityForPeriod": 5
-    },
-    "adjustedGroupInterest": {
-      "qngie": 100,
-      "groupEBITDA": 200,
-      "groupRatio": 50
-    },
-    "declaration": true
-  }
-```
-
-#### Success Response
-
-**Code:** `200 (OK)`
-
-**Response Schema**
-```json
-{
-    "type": "object",
-    "properties": {
-        "acknowledgementReference": {
-            "type": "string"
-        }   
-    }   
-}
-```
-
-**Example Response**
-
-Status: `200 (OK)`
-```json
-{
-    "acknowledgementReference": "ABC1234"
-}
-```
-
-#### Error Responses
-
-**Response Schema**
-```json
-{
-  "type": "array",
-  "minItems": 1,
-  "items": {
-     "type": "object",
-     "properties": {
-       "code": {
-         "type": "string"
-       },
-       "message": {
-         "type": "string"
-       }
-     }
-  } 
-}
-```
-
-**Responses**
-
-| Http Status | Code | Message |
-|-------------|------|---------|
-| 400 | BadRequest | No JSON body received with the request |
-| 400 | BadRequest | Invalid JSON body received. Failed to parse with reason {x} |
-| 400 | BadRequest | Field {x} failed validation rules for reason {y} |
-| 401 | Unauthenticated | Unauthenticated request received |
-| 500 | InternalServerError | Unexpected Internal Server Error |x
-
+## License
 
-**Example response**
+This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
 
-Status: `500 (INTERNAL_SERVER_ERROR)`
-```javascript
-[{
-    "code": "InternalServerError"
-    "message": "Unexpected Internal Server Error"
-}]
-```
+## Viewing Documentation
+### Locally
+- Run Interest Restriction Return and other required services with the script:
 
-## License
+    ```
+     ./run_local_preview_documentation.sh
+    ```
+- Navigate to the preview page at http://localhost:9680/api-documentation/docs/openapi/preview
+- Enter the full URL path of the OpenAPI specification file with the appropriate port and version:
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
+    ```
+     http://localhost:9261/api/conf/1.0/application.yaml
+    ```
+- Ensure to uncomment the lines 60-63 in application.conf, in case of CORS errors
```

Comment:
Add in link to line numbers for accessibility 
`https://github.com/hmrc/individual-benefits/blob/main/conf/application.conf#L60-L63`